test/evp_test.c

```diff
@@ -1074,62 +1102,59 @@ static int cipher_test_run(EVP_TEST *t)
         t->err = "NO_TAG";
         return 0;
     }
-    for (out_misalign = 0; out_misalign <= 1;) {
+
+    for (in_place = 1; in_place >= 0; in_place--) {
         static char aux_err[64];
         t->aux_err = aux_err;
```

Comment:
insert blankline after variables..

Reply:
Code-style related problems should be fixed by now, thanks for review.

Reply:
I missed that, thank you.